src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1526,12 +1526,21 @@ k
         cc.save();
         cc.translate(this.canvas.panX, this.canvas.panY);
 
+        // TODO: Remove the jQuery in favor of _each()!
         $.each(airport.terrain || [], (elevation, terrainLevel) => {
+            if (elevation < 1000) {
+                console.warn(`${airport.icao}.geojson contains 'terrain' at or` +
+                    ' below sea level, which is not supported!');
+
+                // within `$.each()`, this return acts like `continue;`
+                return;
+            }
+
             max_elevation = Math.max(max_elevation, elevation);
-            const color = `rgba(${UiController.terrain.COLOR[elevation]}, `;
+            const color = `hsla(${this.theme.TERRAIN.COLOR[elevation]}`;
 
-            cc.strokeStyle = `${color} ${UiController.terrain.BORDER_OPACITY})`;
-            cc.fillStyle = `${color} ${UiController.terrain.FILL_OPACITY})`;
+            cc.strokeStyle = `${color}, ${this.theme.TERRAIN.BORDER_OPACITY})`;
```

Comment:
```
const generateRgbStr = (hsl, opacity) => `rgba(${rgb}, ${opacity})`;
```
^ We could do this? But that hardly seems worth the abstraction.

And as for the example you gave, that would only be valid for the drawing of the terrain, otherwise, we'd want to pass the opacity rather than directly referencing it in order for it to be more broadly applicable.

Overall, there are two instances of `rgba( ${color}...` and one instance of `hsla( ${color}...`. What would you like to do?